src/tyr/route_serializer_osrm.cc

```diff
@@ -484,6 +485,66 @@ json::ArrayPtr intersections(const valhalla::DirectionsLeg::Maneuver& maneuver,
       }
     }
 
+    // Process turn lanes - which are stored on the previous edge to the node
+    // Check if there is an active turn lane
+    // Verify that turn lanes are not non-directional
+    if (prev_edge && (prev_edge->turn_lanes_size() > 0) && prev_edge->HasActiveTurnLane() &&
+        !prev_edge->HasNonDirectionalTurnLane()) {
```

Comment:
For my understanding - why did you exclude non-directional turn lanes? Is it to avoid having lanes that are all `straight`?

Reply:
@danpaz this is so we do not return turn lanes with `none` or `empty`. If possible, our goal is to determine what they should be and mark them accordingly during data processing - mostly they will be marked as `through`